### DISCUSSION TITLE
Automate your garage door for 5€ - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#49] [Comments]
# Automate your garage door for 5€ - Haade.fr

a simple way to automate your garage door at a lower cost

https://haade.fr/en/blog/domotize-your-garage-door-for-5

**Comment by noseshimself** [comment]
1) How do you know whether the door is open(ing) or closed? If you are using an ESP anyway, using a second input for a reed contact (like the professional packages) would be a good idea as long as you are dealing with a device that has only one button.

2) You could also connect to the emergency stop button so another sensor could cause an emergency stop via Home Assistant. 

3) You don't have to solder anything to the (bloody expensive Molex KK) backup battery connector. There is a block of four terminals just besides the connector for external buttons; the ones labeled 5 and 20 provide 0V and +24V DC:

![image](https://github.com/user-attachments/assets/5fafc29a-0e20-4640-bfdb-33eea786b044)
![image](https://github.com/user-attachments/assets/8297b280-7d39-4073-a1f0-a83d3b5464cc)
A screwdriver is all you need.
